src/components/Navbar.vue

```diff
@@ -36,40 +36,62 @@
           <b-dropdown-item to="/about/team">
             Our Team
           </b-dropdown-item>
-          <b-dropdown-item
-            to="/about/publications">
-            Publications
-          </b-dropdown-item>
 
           <b-dropdown-item href="https://medium.com/@MonarchInit" target="_blank" rel="noopener noreferrer">
-            Blog
+            Disclaimer
           </b-dropdown-item>
 
-          <b-dropdown-item href="https://github.com/monarch-initiative" target="_blank" rel="noopener noreferrer">
-            GitHub Organization
+          <b-dropdown-divider/>
+
+          <b-dropdown-item to="/sources">
+            Data Sources
           </b-dropdown-item>
 
-          <b-dropdown-divider/>
+          <b-dropdown-item to="/sources">
+            Monarch API
+          </b-dropdown-item>
+
+          <b-dropdown-item to="/sources">
+            Releases
+          </b-dropdown-item>
 
           <b-dropdown-item href="https://archive.monarchinitiative.org/latest/" target="_blank" rel="noopener noreferrer">
             Data Downloads
           </b-dropdown-item>
+        </b-nav-item-dropdown>
+
+        <b-nav-item-dropdown text="Documentation">
+          <b-dropdown-item to="/about/publications">
+            Publications
+          </b-dropdown-item>
 
-          <b-dropdown-item to="/sources">
-            Data Sources
+          <b-dropdown-item href="https://github.com/monarch-initiative" target="_blank" rel="noopener noreferrer">
+            GitHub Organization
+          </b-dropdown-item>
+
+          <b-dropdown-item href="https://github.com/monarch-initiative" target="_blank" rel="noopener noreferrer">
```

Comment:
@iimpulse - I know you decided to merge this (given our positive reviews), but there is actually a typo I did not catch earlier. This menu item should actually be "Phenotype Curation Guidelines" - not just 'Guidelines' and (for now) it should link to this blog post http://monarch-initiative.blogspot.com/2015/01/how-to-annotate-patients-phenotypic.html